cli/src/rust.ts

```diff
@@ -1,23 +1,23 @@
 import * as async from './async/child_process';
 import * as child_process from 'child_process';
 
-export type Toolchain = string;
+export type Toolchain = string | null;
```

Comment:
You can remove the `| null` from this definition once you make the change I suggested above.